docs/windows.md

```diff
@@ -206,65 +161,60 @@ sed "s|.*kubeAPIServerOverride: \"\"|    kubeAPIServerOverride: \"${KUBE_APISERV
 kubectl apply -f -
 ```
 
-Since Antrea 1.13, you can deploy both antrea-agent and antrea-ovs Windows DaemonSets
-with containerd runtime by applying file `antrea-windows-containerd-with-ovs.yml`. The
-following commands download the manifest, set kubeAPIServerOverride, and deploy
-the antrea-agent and antrea-ovs Windows DaemonSets with containerd runtime:
-
-```bash
-KUBE_APISERVER=$(kubectl config view -o jsonpath='{.clusters[0].cluster.server}') && \
-curl -sL https://github.com/antrea-io/antrea/releases/download/<TAG>/antrea-windows-containerd-with-ovs.yml | \
-sed "s|.*kubeAPIServerOverride: \"\"|    kubeAPIServerOverride: \"${KUBE_APISERVER}\"|g" | \
-kubectl apply -f -
-```
+When using `antrea-windows-containerd.yml`, you will need to install OVS
+userspace daemons as services when you prepare your Windows worker Nodes, in the
+next section.
 
 #### Join Windows worker Nodes
 
 ##### 1. (Optional) Install OVS (provided by Antrea or your own)
 
-Antrea supports running OVS on Windows as native services or inside a
-host-process container. If you have an OVS package with a signed kernel
-driver and want to run OVS inside container, you can skip this step.
+Depending on which method you are using to install Antrea on Windows, and
+depending on whether you are using your own [signed](https://docs.microsoft.com/en-us/windows-hardware/drivers/install/driver-signing)
+OVS kernel driver or you want to use the test-signed driver provided by Antrea,
+you will need to invoke the `Install-OVS.ps1` script differently (or not at all).
 
-Antrea provides a pre-built OVS package which contains test-signed OVS kernel
-driver. If you don't have a self-signed OVS package and just want to try the
-Antrea on Windows, this package can be used for testing. We also provide a helper
-script `Install-OVS.ps1` to install the OVS driver and register userspace binaries
-as services.
+| Containerized OVS daemons? | Test-signed OVS driver? | Run this command |
```

Comment:
I clarified this section with a table

Reply:
If `Test-signed OVS driver` is no, does it mean user actually uses a signed driver? And it means user is supposed to involve the OVS packages including the driver in the image by themselves if `Containerized OVS daemons` is yes in the meanwhile?

Reply:
> If Test-signed OVS driver is no, does it mean user actually uses a signed driver?

Yes

> And it means user is supposed to involve the OVS packages including the driver in the image by themselves if Containerized OVS daemons is yes in the meanwhile?

I think this is similar to what we discussed with @XinShuYang in https://github.com/antrea-io/antrea/pull/5789/files#r1452075390. I don't think users should be expected to build a custom image. This is why we probably want to document this case more. We should probably suggest that users load their (signed) driver manually on each Node.